### ISSUE TITLE
How to use KSE in fc?

### ISSUE THREAD

**Issue by Alexanzhuo** [yuchaoli/KSE#5]
Thank you for providing this work.
I wonder that how to use kse in nn.Linear? In your paper, you argue that “ It is generic, and can be used to compress fully-connected layers by treating them as 1 × 1 convolutional layers.” 
In your implementation, it seems that you only use KSE for conv_2d?

**Comment by yuchaoli** [comment]
Yes, because of the nn.Linear can be directly transformed into 1 × 1 convolutional layers, so we do not implement the KSE in the FC layer.
